packages/web/app/src/pages/organization-settings.tsx

```diff
@@ -443,6 +451,31 @@ const SettingsPageRenderer = (props: {
             </CardContent>
           </Card>
         )}
+
+        {organization.viewerCanTransferOwnership && (
```

Comment:
TODO @TuvalSimha The field here does not match the action?

You can introduce a new permission in the `authz.ts` file `auditLog:export` and add a `Organization.viewerCanExportAuditLogs` GraphQL field to use instead.